### ISSUE TITLE
Mapping between kb entity id and text

### ISSUE THREAD

**Issue by shmsw25** [xwhan/Knowledge-Aware-Reader#1]
Hello, thanks a lot for releasing the code for the paper. It is very helpful. I am wondering where can I find the mapping between the kb entity (e.g. "m.16jpgj") to a real text. I believe it is needed to obtain glove embeddings of KB entities, which are part of the released preprocessed data.

I have looked at several different sources and it looks like people have been using the Freebase from [FastRDFStore Package](https://github.com/scottyih/FastRDFStore). I have downloaded Freebase dump from this repo, but it looks like 17% of KB entities in your preprocessed data are missing.

I would really appreciate if you can provide a pointer to the public data with the mapping, or release that mapping for WebQSP. Thanks!

**Comment by xwhan** [comment]
Hi, I encountered similar mismatch issues when trying to set up sparql freebase servers. For this project, I ended up using the same entity embeddings released by the GraftNet paper. For the names of entities, you might probably use this mapping [http://nlp.cs.ucsb.edu/data/mi2name](http://nlp.cs.ucsb.edu/data/mi2name) I extracted earlier from Freebase. But not sure whether it covered everything in WebQSP~
